### PR TITLE
Build: Include minified version of additional methods in npm package

### PR DESCRIPTION
**Pull request by Arkni**

### CODE REVIEW

**Comment by Arkni** [comment]
@staabm 

Should we also add the `localization` folder to tags? In other words, adding it to this list: https://github.com/jquery-validation/jquery-validation/blob/5482dfd95ff01bb89a4fe5171f67c63d90c12198/build/release.js#L39-L44

**Comment by staabm** [comment]
Since nobody requested localizations in the npm package I would leave them out for now.

Whats your opinion?

**Comment by Arkni** [comment]
The localization folder is already distributed in the npm package, see https://github.com/jquery-validation/jquery-validation/blob/5482dfd95ff01bb89a4fe5171f67c63d90c12198/package.json#L30-L31

I'm talking about the dist folder committed to GitHub as part of the released tag. See [the dist folder](https://github.com/jquery-validation/jquery-validation/tree/1.17.0/dist) of the release `1.17.0` as an example.

**Comment by staabm** [comment]
oh I see. please add it then.

File: build/release.js [hunk 1]
```diff
@@ -37,6 +37,7 @@ Release.define({
 	generateArtifacts: function( done ) {
 		Release.exec( "grunt release", "Grunt command failed" );
 		done([
```
Comment (staabm) [inline build/release.js]: sorry for beeing late to the party:

please add a note like `keeps this is of files in sync with package.json "files"-key`

Comment (Arkni) [reply build/release.js]: Done!

**Comment by staabm** [comment]
thx!

**Pull request merged**